### DISCUSSION TITLE
Can this theme be used on Windows?

### DISCUSSION THREAD

**Discussion by MrPancakes39** [rafaelmardojai/firefox-gnome-theme#447] [Q&A]
So I love the theme and use it on my Linux but I was hoping I could also use it on my Windows.

**Comment by rafaelmardojai** [comment]
I haven't tested it on Windows, but since the theme uses some Linux only things like `-moz-gtk-csd-minimize-button`, I guess that it will look partially broken.

**Reply by MrPancakes39** [reply]
Interesting, thanks for the reply! I haven't had time to try it out on Windows yet. I'll try it asap and tell you my result.
